[2022-12-28T04:38:23.481+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bertopic_autopik_airflow.create_bento manual__2022-12-28T04:30:31.079084+00:00 [queued]>
[2022-12-28T04:38:23.501+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bertopic_autopik_airflow.create_bento manual__2022-12-28T04:30:31.079084+00:00 [queued]>
[2022-12-28T04:38:23.502+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T04:38:23.502+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-28T04:38:23.502+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T04:38:23.534+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): create_bento> on 2022-12-28 04:30:31.079084+00:00
[2022-12-28T04:38:23.542+0000] {standard_task_runner.py:54} INFO - Started process 541 to run task
[2022-12-28T04:38:23.545+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bertopic_autopik_***', 'create_bento', 'manual__2022-12-28T04:30:31.079084+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/bertopic_autopik_***.py', '--cfg-path', '/tmp/tmprwobwhge']
[2022-12-28T04:38:23.546+0000] {standard_task_runner.py:83} INFO - Job 13: Subtask create_bento
[2022-12-28T04:38:23.547+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/bertopic_autopik_***.py
[2022-12-28T04:38:36.124+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-12-28T04:38:36.215+0000] {task_command.py:384} INFO - Running <TaskInstance: bertopic_autopik_airflow.create_bento manual__2022-12-28T04:30:31.079084+00:00 [running]> on host 2f95ab99defd
[2022-12-28T04:38:36.325+0000] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=hojun.seo@pikurate.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bertopic_autopik_***
AIRFLOW_CTX_TASK_ID=create_bento
AIRFLOW_CTX_EXECUTION_DATE=2022-12-28T04:30:31.079084+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-28T04:30:31.079084+00:00
[2022-12-28T04:38:36.326+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-28T04:38:36.326+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', ' \n            cd /opt/***/dags/bentoml/auto_pik_bento; bentoml build\n        ']
[2022-12-28T04:38:36.372+0000] {subprocess.py:86} INFO - Output:
[2022-12-28T04:38:37.947+0000] {subprocess.py:93} INFO - Error: [bentoml-cli] `build` failed: Failed to import module "bento_service_auto_pik_local.py": No module named 'bento_service_auto_pik_local'
[2022-12-28T04:38:38.104+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2022-12-28T04:38:38.123+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-12-28T04:38:38.133+0000] {taskinstance.py:1401} INFO - Marking task as FAILED. dag_id=bertopic_autopik_***, task_id=create_bento, execution_date=20221228T043031, start_date=20221228T043823, end_date=20221228T043838
[2022-12-28T04:38:38.510+0000] {standard_task_runner.py:102} ERROR - Failed to execute job 13 for task create_bento (Bash command failed. The command returned a non-zero exit code 1.; 541)
[2022-12-28T04:38:38.609+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-12-28T04:38:38.659+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
